rails_exercises/ecommerce_front/admin/app/controllers/admin/feature_products_controller.rb

```diff
@@ -0,0 +1,57 @@
+require_dependency "admin/application_controller"
+
+module Admin
+  class FeatureProductsController < ApplicationController
+    before_filter :set_feature_product, only: [:edit, :update, :destroy]
+
+    def index
+      @feature_products = FeatureProduct.all
+    end
+
+    def new
+      @feature_product = FeatureProduct.new
+    end
+
+    def create
+      @feature_product = FeatureProduct.new(feature_product_params)
+
+      if @feature_product.save
+        redirect_to action: 'index'
+      else
+        render :new
+      end
+    end
+
+    def edit; end
+
+    def update
+      @feature_product.update_attributes(feature_product_params)
+
+      if @feature_product.save
```

Comment:
`update_attributes` already saves the record.
If you want to save separately you can use `assign_attributes`